### DISCUSSION TITLE
Play a hurt sound when taking damage

### DISCUSSION THREAD

**Discussion by ShadowCommander** [space-wizards/space-station-14#3972] [Ideaguys]
Create or find a hurt sound and make it play when a human takes damage.

**Comment by ghost** [comment]
tried looking into this but there doesn't seem to be a generic OnDamage event that I could hook into, I might've missed it though.

**Comment by siyengar04** [comment]
I don't know about adding it in, but I'd say maybe use the Minecraft "oauagh" sound, or a simpler one from soundsnap: https://www.soundsnap.com/tags/damage. If we use a scream or vocally-produced sound, we can adjust pitch and tone per race. Or, if we want a "simpler", more "modern" approach, a "ding" sound might work, like the TF2 crit sound. 

Other than that, like @Qustinnus mentioned, we would need to set up an event that triggers when the player is damaged.

**Reply by Baa14453** [reply]
the minecraft callback with this dumb sound would be great, space station has always been full of random references like that

Might be jarring if it was the only sound, so could have a bank of sounds that play randomly idk

**Comment by Peptide90** [comment]
1 year old issue. Scream sound and action is in game so might look into this.
